### DISCUSSION TITLE
How to load .vtk files?

### DISCUSSION THREAD

**Discussion by blackears** [tkeskita/BVtkNodes#96] [Q&A] [answered]
I was asked to load a .vtk file in Blender.  I was able to get our addon set up and was even able to render the test head you post as an example.  But I am unable to load any .vtk files.  

The example has you load a .vki file which looks like an XML version of a vtk.  However, vtkXMLImageDataReader failes with .vtk files.  vtkImageReader and vtkImageReader2 also fail.  Is there a node that lets you read vtk files?

**Comment by tkeskita** [comment, marked as answer]
Hi, *vtkPolyDataReader* can read point and surface data from .vtk files.
